portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body id="ravenclaw_stripes">
+    <div id="content">
+        <h1 id="title"><strong>Alisa's Gallery</strong></h1>
+        <div id="section">
+            <h2>Gallery</h2>
+            <div id="images">
+                <p><strong>My Taekwondo Team</strong></p>
+                <div id="image_label">
+                    <a href="/images/TKDCornellTeam.jpg">
+                        <img src="/images/TKDCornellTeam.jpg" width=250 alt="Cornell TKD Team Picture">
+                    </a>
+                    <p>This is my Taekwondo team!</p>
+                </div>
+                <div id="image_label">
```

Comment:
use hyphen as delimiter for ids and classes:
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
Added a commit which changes the delimiters to hyphens.